README.md

```diff
@@ -63,38 +63,48 @@ When you run the pipeline, you will use the mandatory `--input` parameter to spe
 
-### 3. Download files
-#### Download the sub-interval files required for Mutect2 
-  - **Step 1**: Click on the [link](https://console.cloud.google.com/storage/browser/genomics-public-data/resources/broad/hg38/v0;tab=objects?pli=1&prefix=&forceOnObjectsSortingFiltering=false&pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22)))
-  - **Step 2**: Click on [scattered_calling_intervals] and select all checkboxes and click `Download`. 
-  - **Step 3**: To download all the files, you will need to install the utility `gsutil`. Please follow the steps as shown below in a command-line window
-    - Download, unzip and install the excecutable  
-      - curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-457.0.0-linux-x86_64.tar.gz
-      - tar -xf google-cloud-cli-457.0.0-linux-x86_64.tar.gz
-      - ./google-cloud-sdk/install.sh
-      - Close rhe current command-line window and re-open it. The utility `gsutil` will now be available in the path.
-  - **Step 4**:Go back to Step (2) and download the files as suggested.
 
-#### A Panel of Normals (PoN) for Mutect2  
-The user can create a PoN file using our [pipeline](https://github.com/Sydney-Informatics-Hub/Somatic-ShortV?tab=readme-ov-file#4-create-pon-per-genomic-interval).
+#### Create a Panel of Normals (PoN) for Mutect2  
+The user can create a a panel of normals (PoN) containing germline and artifactual sites for use with Mutect2 using the instructions provided [here](https://gatk.broadinstitute.org/hc/en-us/articles/360037058172-CreateSomaticPanelOfNormals-BETA-). We will include this functionality as an optional module in the next version of the pipeline.
 
-#### Common biallelic variant resources for GetPileupSummaries 
-The user can create the Common biallelic variant resource files using our [pipeline](https://github.com/Sydney-Informatics-Hub/Somatic-ShortV?tab=readme-ov-file#0-optional-create-common-biallelic-variant-resources)
 
-### 4. Clone this repository 
+#### Create common biallelic variant resources for GetPileupSummaries 
+- A bash script `gatk4_selectvariants.pbs` is provided in the `scripts` folder which can read in your public resource VCF, and select common biallelic SNP variants (by default, those with an allele frequency of > 0.05).
+- This script can be directly run on the NCI Gadi HPC but it can also be modified to run on any other computational resource.
+- In the `gatk4_selectvariants.pbs` script, you can replace `<>` with your resource for resource=`<path/to/public_dataset.vcf.gz>` and output file resource_common=`</path/to/output_public_dataset_common_biallelic.vcf.gz>`. 
+- On NCI Gadi HPC you can adjust the compute resouces and submit your job using the command :`qsub gatk4_selectvariants.pbs`.
+- We will include this functionality as an optional module in the next version of the pipeline.
```

Comment:
Why is this in dot points when other sections aren't? Please be more consistent with your formatting and try format things in a way that is easy to read for your user.